### PR TITLE
[REPO-5622] - Fix Jackson deserialization

### PR DESCRIPTION
**Pull request by nazarethjim**

### CODE REVIEW

File: alfresco-java-event-api/alfresco-java-event-api-integration/src/main/java/org/alfresco/event/sdk/integration/transformer/EventGenericTransformer.java [hunk 1]
```diff
@@ -36,19 +37,9 @@
...
-     *
-     * @param objectMapper given @{@link ObjectMapper} to use in the JSON transformation process
-     */
-    public EventGenericTransformer(final ObjectMapper objectMapper) {
-        this.objectMapper = Objects.requireNonNull(objectMapper);
-    }
-
     @Override
     public RepoEvent<DataAttributes<Resource>> transform(final String eventJSON) {
+        final ObjectMapper objectMapper = new ObjectMapperFactory().createObjectMapper();
```
Comment (zoltanpalfi) [inline alfresco-java-event-api/alfresco-java-event-api-integration/src/main/java/org/alfresco/event/sdk/integration/transformer/EventGenericTransformer.java]: It's fine to remove it from the constructor parameter, but let's leave it as a field that we either initialise in the constructor or in the definition. I agree on not having it as a static as that can end up as a bottleneck, but let's create one per object vs one per call as it is now.

**Review by zoltanpalfi — CHANGES_REQUESTED** [review]
Looks good! Only one small change request.

Now that the project is a public repo, not everyone contributing has access to the JIRA issue. Please let's create an issue describing the problem, so there is better visibility on why we need these changes, and instead of the JIRA reference, let's use the issue reference in the PR.

File: alfresco-java-event-api/alfresco-java-event-api-integration/src/main/java/org/alfresco/event/sdk/integration/transformer/EventGenericTransformer.java (hunk 1 above)
Comment (nazarethjim) [reply alfresco-java-event-api/alfresco-java-event-api-integration/src/main/java/org/alfresco/event/sdk/integration/transformer/EventGenericTransformer.java]: I agree. Changed!

**Review by zoltanpalfi — APPROVED** [review]

**Comment by zoltanpalfi** [comment]
#54 Was created, linking PR

**Pull request merged**